Cargo.toml

```diff
@@ -12,9 +12,12 @@ crate-type = [
 ]
 
 [dependencies]
+getrandom = "0.2.15"
 log = "0.4.22"
 pyo3 = "0.22.4"
 pyo3-log = "0.11.0"
+rand_core = "0.6.4"
```

Comment:
See above

Reply:
Dropped those unneeded dependencies in favor of `rand` by using:

```rust
StdRng::from_entropy().fill_bytes(&mut random_data);
```